tests/playbooks/fedora32_deps.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install CentOS 8 deps for network system role tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+        - git
+        - python3-pip
+  tasks:
+    - name: Install EPEL
+      package:
+        name: epel-release
+        state: present
```

Comment:
This will not work on Fedora.

Reply:
You are right. It is also unnecessary for the given distribution version.